src/Mutator/Extensions/BCMath.php

```diff
@@ -106,46 +107,71 @@ public function canMutate(Node $node): bool
-                $this->makePowerModuloMapper()
-            )),
-            'bccomp' => $this->makeCheckingMinArgsMapper(2,
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Spaceship::class)
-            ),
-        ];
-
-        $functionsToRemove = array_filter($functionsMap, static function (bool $isOn): bool {
-            return !$isOn;
-        });
-
-        $this->converters = array_diff_key($converters, $functionsToRemove);
+        return array_intersect_key(
+            [
+                'bcadd' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Plus::class)
+                    )
+                ),
+                'bcdiv' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Div::class)
+                    )
+                ),
+                'bcmod' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Mod::class)
+                    )
+                ),
+                'bcmul' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Mul::class)
+                    )
+                ),
+                'bcpow' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Pow::class)
+                    )
+                ),
+                'bcsub' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeCastToStringMapper(
+                        self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Minus::class)
+                    )
+                ),
+                'bcsqrt' => self::makeCheckingMinArgsMapper(
+                    1,
+                    self::makeCastToStringMapper(
+                        self::makeSquareRootsMapper()
+                    )
+                ),
+                'bcpowmod' => self::makeCheckingMinArgsMapper(
+                    3,
+                    self::makeCastToStringMapper(
+                        self::makePowerModuloMapper()
+                    )
+                ),
+                'bccomp' => self::makeCheckingMinArgsMapper(
+                    2,
+                    self::makeBinaryOperatorMapper(Node\Expr\BinaryOp\Spaceship::class)
+                ),
+            ],
+            array_fill_keys($functionsMap, null)
+        );
     }
 
-    private function makeCheckingMinArgsMapper(int $minimumArgsCount, callable $converter)
```

Comment:
`callable` is a far more tricky type to handle than `Closure` hence if we can, I think `Closure` is better